packages/vitepress-theme/package.json

```diff
@@ -34,5 +34,8 @@
   "dependencies": {
     "@fontsource-variable/source-code-pro": "^5.1.0",
     "@fontsource-variable/source-sans-3": "^5.1.0"
+  },
+  "devDependencies": {
+    "sit-onyx": "workspace:^"
```

Comment:
onyx is already a peerDependency, why is it added as devDependency as well?

Reply:
- updated turbo to determine dependencies itself

    -  duplicated peerDependencies from the workspace into devDependencies as turbo will not consider them otherwise (See also: vercel/turborepo@1eb3333)